packages/validator-ajv8/src/precompiledValidator.ts

```diff
@@ -156,7 +156,12 @@ export default class AJV8PrecompiledValidator<
         'The schema associated with the precompiled validator differs from the rootSchema provided for validation'
       );
     }
-    const validator = this.getValidator(schema);
-    return validator(formData);
+    try {
+      const validator = this.getValidator(schema);
+      return validator(formData);
+    } catch (e) {
+      console.warn('Error encountered getting validator', e);
+      return false;
+    }
```

Comment:
While this is a decent fix, I really do want things to blow up. I would recommend, instead, adding an well known id to the junk schema (i.e. create a constant for the junk schema id in the `utils/src/constants.ts` file) and adding a check here to just return false for that id. Does that make sense?

Reply:
Yeah that makes sense!